addon/build-waiter.ts

```diff
@@ -1,5 +1,6 @@
+import { AsyncTestWaiter, TestWaiter } from 'ember-test-waiters';
```

Comment:
I'd love it if we just move the class directly into here, simplifies the "mental model" and helps us avoid "exposing" the class when we don't really want to.

Reply:
Agree.

Reply:
Done.